std/machines/hash/poseidon_gl_memory.asm

```diff
@@ -11,9 +11,9 @@ use std::machines::split::split_gl::SplitGL;
 // This version of the Poseidon machine receives memory pointers and interacts
 // with memory directly to fetch its inputs and write its outputs.
 // In comparison to std::machines::hash::poseidon_gl::PoseidonGL, this machine has:
-// - 18 extra witness columns:
+// - 20 extra witness columns:
```

Comment:
Is it worth it 2 more columns to avoid a 96 bytes copy?

Well, probably, otherwise you wouldn't have opened the PR.

Reply:
I am not too sure it's worth it! The motivation is that it helps in the hot path of the bootloader: Each word that is paged-in is written to memory anyway, but if the inputs need to be consecutive, it needs to be written to another memory location as well.

My intuition is that either way is fine and I'm also happy to discard this PR. But we can try and measure the effect.